spec/requests/api/v1/buisness_intelligence_request_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe "Buisness Intelligence Endpoints", type: :request do
+  it "can return the total revenue for a single merchant" do
```

Comment:
Was important to set up a bunch of different relationships here to get this test to pass.